### ISSUE TITLE
create `pid_to_eml_datatable` helper

### ISSUE THREAD

**Issue by jeanetteclark** [NCEAS/arcticdatautils#59]
similar to `pid_to_eml_other_entity`, this function will use sysmeta to generate a physical section for a dataTable object, in addition to inserting attributes and factors.

arguments

- mn = member node instance

- pid = pid of object published on that member node

- attributes = data frame of attributes with columns attributeName, attributeDefinition, measurementScale, domain, formatString, definition, unit, numberType, missingValueCode, missingValueCodeExplanation

- factors = data frame of factors with column names attributeName, code, definition. This argument is not required for the function to run

this function would need to call `pid_to_eml_physical` (which is a wrapper around `sysmeta_to_eml_physical`) and is part of `arcticdatautils` in addition to `set_attributes` (which is the function that will take the attributes and factors dataframes as arguments)

it might be nice if we also incorporate an attribute validation check within this function

**Comment by amoeba** [comment]
Thanks for filing this, @jeanetteclark. That all sounds doable and awesome. Marked as a good first issue for any intrepid contributors. 

This in particular is cool:

> it might be nice if we also incorporate an attribute validation check within this function

`arcticdatautils::eml_validate_attributes` is decently fast so I'd say we do this!

**Comment by emilyodean** [comment]
I'll take this one.

**Comment by amoeba** [comment]
Awesome, thanks @emilyodean !

**Comment by isteves** [comment]
@emilyodean I tried this function out yesterday.  I had two pids with the same data table, so I passed both into the `pid` argument and ended up with one data table with 2 physicals.  I'm kind of a fan of this, but I think we generally want to have separate data tables for each physical.  If you plan to expand this function even more, it might be a direction to consider!  (and perhaps keep both options open)
